### ISSUE TITLE
Rename Specific PBS Material Map Fields

### ISSUE THREAD

**Issue by H3BO3** [Frooxius/NeosPublic#1743]
## Is your tweak request related to a problem? Please describe.
Confusion regarding users setting up PBR materials manually, especially in regards to the field for the Metallic/Specular and Smoothness map

## Describe what would you like tweaked
Rename the relevant PBS Metallic/Specular map fields to MetallicSmoothness/SpecularSmoothness.

## Describe alternatives you've considered
I can't think of much else, unless some kind of descriptor or hint is included with fields that describes what they're for, but that seems more like a documentation thing. The described tweak would be much simpler

## Additional context
Confusion from Beef Taquitos#1242 when attempting to set up PBS materials.

**Comment by shadowpanther** [comment]
The field names that would make more sense would me "Metallicity/Smoothness" and "Specular(Specularity?)/Smoothness", so users wouldn't confuse those with _just_ the smoothness map for the metallic/specular workflow respectively.

**Comment by H3BO3** [comment]
The map names are combined like that in filenames, so that's why I didn't include the slashes.

**Comment by H3BO3** [comment]
Not to mention a slash typically indicates some alternative functionality, rather than a combination of the two.

**Comment by StarfishHidari** [comment]
This would be a helpful change, I think a lot of people only realize that field doubles for both smoothness and metallic/specular is once someone shows them the wiki page on importing. Renaming it like this could help make it more obvious

**Comment by Aerizeon** [comment]
Honestly, this is one of those fields where the inline documentation/tooltips would be useful, since it's kinda unexpected that the alpha channel would have meaning.

**Comment by shadowpanther** [comment]
That's fair, @H3BO3, but we need to somehow communicate that that one field stands for both maps, not make users look for a separate "Metallicity" field instead. "Metallicity and Smoothness"?

**Comment by H3BO3** [comment]
@Aerizeon It still indicates two channels of information. Combined maps from substance painter have that exact same format, so it's more or less standardized. When I first came into Neos, I was also confused at the fact that the field was just named Metallic or Specular even though it was two channels in one, I just never thought about it again until now.

**Comment by H3BO3** [comment]
@shadowpanther I'm not sure what you're saying. My new name suggestion already has "metallic/specular" in it like the current implementation, now it just has "smoothness" along with it too.

**Comment by shadowpanther** [comment]
@H3BO3 when I read "MetallicSmoothness", my first thought is "a smoothness map for a metallic material... so where's the metallicity?".

**Comment by Frooxius** [comment]
Yeah this could be changed. MetallicGlossMap and SpecularGlossMap would be the correct names here, since the field names in the inspector reflect the names in the code and those follow this kind of naming convention. Anything different/more descriptive will require the documentation system instead.

**Comment by H3BO3** [comment]
@shadowpanther The material is already using the metallic or specular workflow as indicated in the shader name. Substance Painter exports the map as MetallicSmoothness/SpecularSmoothness, as I indicated earlier. This is more or less standard PBR lingo

**Comment by H3BO3** [comment]
@Frooxius Forgive me if I'm wrong, but doesn't the Slider for changing the values without a map use the name "Smoothness" instead of "Gloss"? I had that in mind when writing this issue. Not to mention Substance uses the same wording "MetallicSmoothness" in the exported texture name for example

**Comment by Frooxius** [comment]
@H3BO3 Yeah that might actually work better. Internally the maps are called MetallicGlossMap in the shaders and Xiexe uses this as well, but the Smoothness we renamed (even though this one is called "Glossiness" internally). Might be better to rename the Xiexe one to follow this convention as well.

MetallicSmoothnessMap / SpecularSmoothnessMap should be most consistent with the rest of the naming convention then.
